qt_ui/windows/newgame/WizardPages/QFactionSelection.py

```diff
@@ -380,10 +500,68 @@ def _filter_selected_units(qfu: QFactionUnits) -> Faction:
         qfu.updateFactionUnits(fac.missiles)
         return fac
 
+    def show_save_faction_dialog(self, faction: Faction):
+        dialog = QFactionSaver(faction)
+        dialog.exec_()
```

Comment:
Prefer `exec` instead, `exec_` is an old thing because `exec` is also a Python function...